en/SUMMARY.md

```diff
@@ -388,10 +388,13 @@
   - [Helicopter (experimental)](frames_helicopter/index.md)
     - [Helicopter Config/Tuning](config_heli/index.md)
   - [Rovers (experimental)](frames_rover/index.md)
-    - [Drive Modes](flight_modes_rover/index.md)
-    - [Differential-steering Rover](frames_rover/differential_rover.md)
+    - [Differential Rover](frames_rover/differential.md)
+      - [Configuration/Tuning](config_rover/differential.md)
+      - [Drive Modes (DR)](flight_modes_rover/differential.md)
```

Comment:
I'm moving the Drive modes first. This is the pattern everywhere else "Features", "Assembly", "Configuration", and drive modes are a feature.